File: pset6/mario.py
import cs50

while True:
    print("Choose the height (between 0 and 23): ", end="")
    height = cs50.get_int()
    if(height>0 and height<23):
        break
for i in range(height):
    for x in range(height-i):
        print(" ", end = "")
    for a in range(i+1):
        print("#", end = "")
    print("")

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while(height < 0 || height>23);
    
    for (int i = 1; i <= height;i++)
    {
        int rownum = i;
        for(int x = 1; x <= height-rownum; x++){
            printf(" ");
        }
        for (int a = 1; a <= rownum+1; a++){
            printf("#");
        }
        printf("\n");
    }
  
    

}